backend/app.js

```diff
@@ -68,6 +71,37 @@ const projectTeamMembersRouter = require("./routers/projectTeamMembers.router");
 const slackRouter = require("./routers/slack.router");
 const authRouter = require("./routers/auth.router");
 
+// Check that clients to the API are sending the custom request header on all methods
+// except for ones described in the dontCheckCustomRequestHeaderApis array
+app.use(function customHeaderCheck (req, res, next) {
+
+  // console.log(`Path: ${  req.path  } Method: ${  req.method}`);
+
+  let pathToCheck = req.path;
+
+  if(pathToCheck.endsWith("/")){
+    pathToCheck = pathToCheck.substr(0, pathToCheck.length-1);
+  }
+
+  const key = `${req.method}::${pathToCheck}`;
+  // console.log(`key: ${  key  }`);
+  if(!dontCheckCustomRequestHeaderApis.includes(key)) 
+  {
+    // console.log(`CUSTOM HEADER CHECKING FOR KEY: ${key}`);
+    const { headers } = req;
+    const expectedHeader = process.env.CUSTOM_REQUEST_HEADER;
+
+    if (headers[customRequestHeaderName] !== expectedHeader) {
+      // console.log("CUSTOM HEADER NOT FOUND");
```

Comment:
Please uncomment this console output and change the message to the following:
`REQUEST SHOULD CONTAIN CUSTOM HEADER BUT IT ISN'T FOUND`
It will help developers to find missed header easier.